From 54de974f4fce401205f60d30cd56fe7b8c9c4aad Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 8 Aug 2017 14:35:42 +0800
Subject: [PATCH 2351/5242] MLK-16158-6 ARM64: dts: imx8mq-evk: set default
 display mode for adv7535 to 1080p60Hz

commit  727d90ffe466368a01e8cd5522a46911b84e0d70 from
https://source.codeaurora.org/external/imx/linux-imx.git

Set the default display mode for ADV7535 to 1920x1080p@60Hz which
is the maximum resolution can be supported by ADV7535.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 3d91400..562c7e4 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -413,7 +413,7 @@
 		compatible = "adi,adv7535";
 		reg = <0x3d>;		/* PD pin is low */
 		/* TODO: pin config & irq */
-		video-mode = <34>;	/* 1920x1080@30HZ */
+		video-mode = <16>;	/* 1920x1080@60HZ */
 		dsi-traffic-mode = <0>;
 		bpp = <24>;
 		status = "okay";
-- 
1.7.9.5

